### PR TITLE
Fixed Wizard validation is run in readOnly mode

### PR DESCRIPTION
**Pull request by alexandraRamanenka**

### CODE REVIEW

**Comment by travist** [comment]
Please write a test.

File: src/Wizard.js [hunk 1]
```diff
@@ -319,6 +316,14 @@ export default class Wizard extends Webform {
     return NativePromise.reject('Page not found');
   }
 
+  goToPage(index) {
```
Comment (ababic) [inline src/Wizard.js]: I feel that having both a `setPage` and `goToPage` method may cause a little confusion going forward. Perhaps we could add an additional argument to `setPage` to toggle this behaviour instead, perhaps with a `false` default value?

Comment (randallknutson) [reply src/Wizard.js]: Agreed. We should try to not duplicate nearly identical functions.

**Comment by travist** [comment]
Looks good. Just waiting on some tests to ensure this continues to work in the future.

**Pull request merged**